cassandra_test.go

```diff
@@ -1907,6 +1909,9 @@ func TestBatchObserve(t *testing.T) {
 		if stmt != fmt.Sprintf(`INSERT INTO batch_observe_table (id,other) VALUES (?,%d)`, i) {
 			t.Fatal("unexpected query", stmt)
 		}
+		if observedBatch.observedValues[i] != []interface{}{i} {
```

Comment:
```
./cassandra_test.go:1912:38: invalid operation: observedBatch.observedValues[i] != []interface {} literal (slice can only be compared to nil)
./cassandra_test.go:1913:81: observedBatch.observedEntries undefined (type *observation has no field or method observedEntries)
```

`observedBatch.observedEntries` should be `observedBatch.observedValues` instead. But both lines could be replaced with a call to `assertDeepEqual(t, "observed value", []interface{}{i}, observedBatch.observedValues[i]`

https://github.com/gocql/gocql/blob/87a5d7a5ff74ffa9eb716f05f6e26191836bf03d/common_test.go#L263-L268

Reply:
@martin-sucha Fixed the tests. Thanks for suggesting the fix